trestle/transforms/implementations/tanium.py

```diff
@@ -494,21 +487,76 @@ def analysis(self) -> List[str]:
         analysis.append(f'observations: {len(self.observations)}')
         return analysis
 
+    def _get_local_definitions(self, system_component: SystemComponent) -> LocalDefinitions1:
+        """Get local definitions."""
+        rval = LocalDefinitions1()
+        for component in self.components:
+            if component.uuid == system_component.uuid:
+                rval.components = [component]
+                rval.inventory_items = []
+                for inventory_item in self.inventory:
+                    for implemented_component in inventory_item.implemented_components:
+                        if implemented_component.component_uuid == system_component.uuid:
+                            rval.inventory_items.append(inventory_item)
+                break
+        return rval
+
```

Comment:
Is it ok  if this returns an rval that didn't get filled with info?

Reply:
This is allowed, but not desired. Should not cause an exception. Fixed.